pkg/apis/core/validation/validation.go

```diff
@@ -2873,14 +2869,6 @@ func validatePodDNSConfig(dnsConfig *core.PodDNSConfig, dnsPolicy *core.DNSPolic
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("nameservers").Index(i), ns, "must be valid IP address"))
 			}
 		}
-		// Validate searches.
-		if len(dnsConfig.Searches) > MaxDNSSearchPaths {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("searches"), dnsConfig.Searches, fmt.Sprintf("must not have more than %v search paths", MaxDNSSearchPaths)))
-		}
-		// Include the space between search paths.
-		if len(strings.Join(dnsConfig.Searches, " ")) > MaxDNSSearchListChars {
```

Comment:
you cannot remove a max length check.  You can change what that limit is, but we must still control the size of these fields.

Reply:
Okay that's fair. Are there guidelines for limits on config fields or should I just pick something reasonable (maybe 8192 bytes)?